build/template-build-and-run-all-tests.yaml

```diff
@@ -74,10 +74,9 @@ jobs: #Build and stage projects
       checkCoverage: '$(CodeCoverageGateEnabled)'
       coverageFailOption: 'build'
       coverageType: 'lines'
-      allowCoverageVariance: true 
-      coverageVariance: 1 # Specify by how much the current amount of covered or uncovered code (depending on the parameter *Use Uncovered Elements*) may deviate from the previous value before the policy fails. Please be aware that the code coverage may slowly but steadily decrease from build to build if you allow a code coverage variance. Thus, you should keep this value as low as possible.
+      allowCoverageVariance: false 
```

Comment:
With this now the build won't fail but will give it in warning?

Reply:
It will still fail, but it will be a lot more forgiving to non product code changes